src/app/shared/report-header/report-header.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="report-header">
+  <h1 >{{reportTitle}}</h1>
+  <div class="report-header-select">
+    <ion-select style="color: black;" aria-label="Program" interface="popover" placeholder="Select" [multiple]="false" [(ngModel)]="reportType" (ionChange)="getReportType(reportType)">
+      <ion-select-option value="0">Weekly</ion-select-option>
```

Comment:
Can u add these options in a config file and then use it here

Reply:
done